java.lang.ArrayIndexOutOfBoundsException at telemetry ping

java.lang.RuntimeException: 
 
  at android.os.AsyncTask$3.done (AsyncTask.java:318)
 
  at java.util.concurrent.FutureTask.finishCompletion (FutureTask.java:354)
 
  at java.util.concurrent.FutureTask.setException (FutureTask.java:223)
 
  at java.util.concurrent.FutureTask.run (FutureTask.java:242)
 
  at android.os.AsyncTask$SerialExecutor$1.run (AsyncTask.java:243)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1133)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at java.lang.Thread.run (Thread.java:760)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 
 
  at com.android.okhttp.okio.Buffer.writeByte (Buffer.java:973)
 
  at com.android.okhttp.internal.Platform.concatLengthPrefixed (Platform.java:130)
 
  at com.android.okhttp.internal.Platform.configureTlsExtensions (Platform.java:84)
 
  at com.android.okhttp.Connection.connectTls (Connection.java:230)
 
  at com.android.okhttp.Connection.connectSocket (Connection.java:199)
 
  at com.android.okhttp.Connection.connect (Connection.java:172)
 
  at com.android.okhttp.Connection.connectAndSetOwner (Connection.java:367)
 
  at com.android.okhttp.OkHttpClient$1.connectAndSetOwner (OkHttpClient.java:130)
 
  at com.android.okhttp.internal.http.HttpEngine.connect (HttpEngine.java:330)
 
  at com.android.okhttp.internal.http.HttpEngine.sendRequest (HttpEngine.java:247)
 
  at com.android.okhttp.internal.huc.HttpURLConnectionImpl.execute (HttpURLConnectionImpl.java:457)
 
  at com.android.okhttp.internal.huc.HttpURLConnectionImpl.connect (HttpURLConnectionImpl.java:126)
 
  at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getOutputStream (HttpURLConnectionImpl.java:257)
 
  at com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.getOutputStream (DelegatingHttpsURLConnection.java:218)
 
  at com.android.okhttp.internal.huc.HttpsURLConnectionImpl.getOutputStream (HttpsURLConnectionImpl.java)
 
  at org.mozilla.telemetry.net.HttpURLConnectionTelemetryClient.upload (HttpURLConnectionTelemetryClient.java:97)
 
  at org.mozilla.telemetry.net.HttpURLConnectionTelemetryClient.uploadPing (HttpURLConnectionTelemetryClient.java:44)
 
  at org.mozilla.telemetry.schedule.jobscheduler.TelemetryJobService$1.onTelemetryPingLoaded (TelemetryJobService.java:149)
 
  at org.mozilla.telemetry.storage.FileTelemetryStorage.process (FileTelemetryStorage.java:71)
 
  at org.mozilla.telemetry.schedule.jobscheduler.TelemetryJobService.performPingUpload (TelemetryJobService.java:145)
 
  at org.mozilla.telemetry.schedule.jobscheduler.TelemetryJobService.uploadPingsInBackground (TelemetryJobService.java:79)
 
  at org.mozilla.telemetry.schedule.jobscheduler.TelemetryJobService$UploadPingsTask.doInBackground (TelemetryJobService.java:50)
 
  at org.mozilla.telemetry.schedule.jobscheduler.TelemetryJobService$UploadPingsTask.doInBackground (TelemetryJobService.java:46)
 
  at android.os.AsyncTask$2.call (AsyncTask.java:304)
 
  at java.util.concurrent.FutureTask.run (FutureTask.java:237)

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -160,7 +160,7 @@ dependencies {
         transitive = false
     }
 
-    compile 'org.mozilla.telemetry:telemetry:1.0.0'
+    compile 'org.mozilla.telemetry:telemetry:1.1.1'
 
     compile 'com.adjust.sdk:adjust-android:4.11.4'
     compile 'com.google.android.gms:play-services-analytics:10.2.6' // Required by Adjust

